doc/nrf/gs_updating.rst

```diff
@@ -14,6 +14,20 @@ You might also want to switch to a newer release or check out the latest state o
 However, if you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
 For an overview of changes in the latest releases, see :ref:`release_notes`.
 
+.. _gs_updating_in_vsc:
+
+Updating |VSC|
+*****************
```

Comment:
This line must match the length of the header.

Reply:
Fixed